package.json

```diff
@@ -11,6 +11,7 @@
   "node": "dist/verifier-node.js",
   "typings": "dist/index.d.ts",
   "scripts": {
+    "postinstall": "rm -rf node_modules/jsonld-checker/node_modules/jsonld && rm -rf node_modules/@bloomprotocol/ecdsa-secp256k1-verification-key-2019/node_modules/base58-universal",
```

Comment:
this is dirty and I will try to address dependency issues at those level

https://github.com/gjgd/jsonld-checker/issues/91
https://github.com/digitalbazaar/base58-universal/pull/7

Reply:
jsonld checker trail: https://github.com/transmute-industries/did-core/pull/49